admin/src/components/DiscussionThreadItemReviewAction/index.tsx

```diff
@@ -9,39 +9,40 @@ import { IconButton } from "@strapi/design-system/IconButton";
 import { useNotification, useOverlayBlocker } from "@strapi/helper-plugin";
 import { ReviewIcon, LockIcon } from "../icons";
 import { Check } from "@strapi/icons";
-import { resolveReport, resolveMultipleReports } from "../../pages/utils/api";
-import { getMessage, handleAPIError } from "../../utils";
-import ReportsReviewTable from "../ReportsReviewTable";
-import ReportsReviewModal from "../ReportsReviewModal";
-import { pluginId } from "../../pluginId";
+import {
+  resolveReport,
+  resolveCommentMultipleReports,
+} from '../../pages/utils/api';
+import {getMessage, handleAPIError} from '../../utils';
+import ReportsReviewTable from '../ReportsReviewTable';
+import ReportsReviewModal from '../ReportsReviewModal';
+import {pluginId} from '../../pluginId';
 
 const DiscussionThreadItemReviewAction = ({
   item,
   isAnyActionLoading: isAnyActionLoading,
   queryToInvalidate,
   areBlockButtonsDisabled,
-  allowedActions: { canModerate, canAccessReports, canReviewReports },
+  allowedActions: {canModerate, canAccessReports, canReviewReports},
   blockItemMutation,
   blockItemThreadMutation,
   onBlockButtonsStateChange,
   onBlockActionClick,
 }) => {
-  const { reports } = item;
+  const {reports} = item;
 
   const [reportsReviewVisible, setReportsReviewVisible] = useState(false);
   const [storedItems, setStoredItems] = useState([]);
   const [selectedItems, setSelectedItems] = useState([]);
 
   const toggleNotification = useNotification();
   const queryClient = useQueryClient();
-  const { lockApp, unlockApp } = useOverlayBlocker();
+  const {lockApp, unlockApp} = useOverlayBlocker();
 
   useEffect(() => {
-    setStoredItems(orderBy(
-      reports,
-      ["resolved", "createdAt"],
-      ["DESC", "DESC"]
-    ));
+    setStoredItems(
```

Comment:
Offtop: I'd add `reports` to dependancy array. Or more idiomatic `useMemo` would be better.